### ISSUE TITLE
Error 422 when sending Circles

### ISSUE THREAD

**Issue by KVuk** [CirclesUBI/circles-myxogastria#182]
We have an issue with sending Circles from @DavidOrah to @Koliba1, keep on getting Error message 422(Symbol(FAILED_REQUEST).  This happens both on desktop and mobile

**Comment by adzialocha** [comment]
Related: https://github.com/CirclesUBI/circles-api/issues/73 and https://github.com/CirclesUBI/safe-relay-service/issues/43

**Comment by triaslucia** [comment]
@llunaCreixent What is the  current state of this?  I'm  asking because its related to New Notification System we are currently designing. 

Questions: 
1_It's 422 always related to transactions problems? 
2_It's possible to solve by refreshing?

**Comment by triaslucia** [comment]
Its possible this will get fixed, once the Relayer is replaced. Which is ready soon.

**Comment by llunaCreixent** [comment]
@triaslucia The 422 error appears when the transaction is invalid, and the only case we have identified is when the steps are not valid. This cannot be solved by refreshing, but we have enabled a feature that the steps are updated when a transfer fails. So the only thing that users can do is try again later (in few minutes).

This issue is meant to be alleviated once we move to the pathfinder2 service (to be deployed in April).
